packages/infra/lib/app-config-stack.ts

```diff
@@ -65,6 +77,10 @@ export function createAppConfigStack(
             enabled: true,
             cxIds: [],
           },
+          [cxsWithIncreasedSandboxLimitFeatureFlag]: {
+            enabled: true,
+            cxIds: [],
```

Comment:
I think we should call this variable something other than cxID given that the patient limit is also stored within it. Maybe cxIdAndPatientLimit